content/helpStrings.ts

```diff
@@ -0,0 +1,18 @@
+export default {
```

Comment:
I remember seeing the source for all of their tooltips - can't seem to find it now but let's file away a task to just bulk-import all of them.

Reply:
there's a task in Asana for adding them